core/trie2/triedb/pathdb/cache_test.go

```diff
@@ -46,7 +46,7 @@ func TestPutAndGetNode(t *testing.T) {
 
 func TestCacheMisses(t *testing.T) {
 	cache := newCleanCache(1024 * 1024)
-	owner := *new(felt.Felt).SetUint64(1234567890)
+	owner := *felt.NewFromUint64[felt.Address](1234567890)
```

Comment:
Please use the right constructors, in this case, you're returning a reference and copying the value of the result. Why not return a value directly with `felt.FromUint64[...](...)`